auth/src/main/java/com/alibaba/nacos/auth/NacosAuthConfig.java

```diff
@@ -43,6 +43,10 @@
 @EnableGlobalMethodSecurity(prePostEnabled = true)
 public class NacosAuthConfig extends WebSecurityConfigurerAdapter {
     
+    public static final String NACOS_IDENTITY_KEY = "identity";
+    
+    public static final String USERNAME_PASSWORD = "username_password_auth";
```

Comment:
Why not use auth.type to find out the plugin?

Reply:
Should I define an enum type?